dbaas_controller.go

```diff
@@ -16,10 +16,3 @@
 
 // Package dbaascontroller is a dummy package.
 package dbaascontroller
-
```

Comment:
we will have conflict because of these changes, let's keep it.

Reply:
done